content/src/service/ServiceImpl.ts

```diff
@@ -286,12 +284,8 @@ export class ServiceImpl implements MetaverseContentService, ClusterDeploymentsS
         return this.deploymentManager.areEntitiesDeployed(repository.deployments, entityIds)
     }
 
-    getLegacyHistory(from?: Timestamp, to?: Timestamp, serverName?: ServerName, offset?: number, limit?: number): Promise<LegacyPartialDeploymentHistory> {
-        return this.historyManager.getHistory(this.repository.deployments, from, to, serverName, offset, limit)
-    }
-
-    getDeployments(filters?: DeploymentFilters, offset?: number, limit?: number, repository: RepositoryTask | Repository = this.repository): Promise<PartialDeploymentHistory<Deployment>> {
-        return repository.taskIf(task => this.deploymentManager.getDeployments(task.deployments, task.content, task.migrationData, filters, offset, limit))
+    getDeployments(filters?: DeploymentFilters, sortCondition?: SortBy, offset?: number, limit?: number, repository: RepositoryTask | Repository = this.repository): Promise<PartialDeploymentHistory<Deployment>> {
```

Comment:
You know what I just thought. We could make this a little easier to use. Because I saw above that we are calling it and sending:
```service.getDeployments({}, undefined, 0, 1)```

It could be better if we changes the method into something like:
```
getDeployments(options?: DeploymentOptions): ... {

}
type DeploymentOptions = {
    filters?: DeploymentFilters, 
    sortBy?: SortBy, 
    offset?: number, 
    limit?: number }
}
```

That way, we can simply call it like this:
```service.getDeployments({ offset: 0, limit: 1})```. No positional arguments needed, and it's clearer

We should do this both for the service, and the `DeploymentsManager`